docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        from_last_update=False,  # set to false to get activations only from the time when alert config was updated (takes precedence over since of both are passed)
</pre><p>You mean set to <code>True</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>